### PR TITLE
Add iOS implementations for MediaPlayer volume and looping setters

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by krisgiesing** [comment]
LGTM

**Comment by eseidelGoogle** [comment]
Disturbingly simple. :)  lgtm!

File: sky/services/media/ios/media_player_impl.mm [hunk 1]
```diff
@@ -58,6 +60,22 @@ - (BOOL)seekTo:(NSTimeInterval)interval {
   return [_player playAtTime:_player.deviceCurrentTime + interval];
 }
 
+- (void)setVolume:(double)volume {
+  if (volume > 1.0) {
```
Comment (Hixie) [inline sky/services/media/ios/media_player_impl.mm]: do we document what range we're accepting on volume anywhere?

Comment (chinmaygarde) [inline sky/services/media/ios/media_player_impl.mm]: media.mojom is based on the Android interface https://github.com/flutter/engine/commit/04ff30d78b8e1613fccfb4e3c9502bc337ad10da#diff-9f13ad3e7c279cc26b97b442f2d9b81aR118 which takes scalars in the 0.0 to 1.0 range

Comment (chinmaygarde) [inline sky/services/media/ios/media_player_impl.mm]: How are mojoms documented?

Comment (Hixie) [inline sky/services/media/ios/media_player_impl.mm]: Good question.

We should document somewhere, maybe for now just as a comment in the mojom, what the range is and what the implementation's responsibilities are when receiving out-of-range values.

**Pull request merged**